ign_ros2_control/src/ign_ros2_control_plugin.cpp

```diff
@@ -87,6 +87,9 @@ class IgnitionROS2ControlPluginPrivate
   std::shared_ptr<controller_manager::ControllerManager>
   controller_manager_{nullptr};
 
+  /// \brief String with the name of the controller manager node 
+  std::string controller_manager_node_ = "controller_manager";
```

Comment:
To me the variable name implies that this should be the Node object itself, not the name of the node. Consider renaming.

Reply:
@Ronoman I agree with you, happy to change it. Thing is, I tried to replicate existing naming: if you look at line 99 of the same file ` std::string robot_description_node_ = "robot_state_publisher";` author is using the same naming

I suggest we should keep both or change both

Reply:
I have renamed to `robot_description_node_name_` for now. I've cleaned up the variable you were pointing out